src/include/utils/sync_guc_name.h

```diff
@@ -124,3 +124,4 @@
 		"gp_add_column_inherits_table_setting",
 		"gp_resgroup_debug_wait_queue",
 		"gp_log_resqueue_priority_sleep_time",
+		"gp_add_partition_inherits_table_setting",
```

Comment:
This file is used to fill the sync_guc_names_array array which is passed to bsearch, so the array should be sorted. @RekGRpth,  Did you test what happens when array is unsorted?

Reply:
Lines 125 and 126 were already unsorted; I simply added them to the end of the list, similar to other additions.

Reply:
> the sync_guc_names_array array which is passed to bsearch

Before using [search](https://github.com/arenadata/gpdb/blob/a37bfcb0fe7402d395272012e239e4ea65e86772/src/backend/utils/misc/guc_gp.c#L5289-L5293), the array is first [sorted](https://github.com/arenadata/gpdb/blob/a37bfcb0fe7402d395272012e239e4ea65e86772/src/backend/utils/misc/guc_gp.c#L5270-L5272).

Reply:
Is this answer sufficient?